### PR TITLE
Removing FMDB from Cordova plugin

### PR DESCRIPTION
**Pull request by khawkins**

### CODE REVIEW

File: tools/update.sh [hunk 1]
```diff
@@ -133,7 +133,7 @@ cd ${ROOT_FOLDER}
 
 echo "*** Creating directories ***"
 echo "Starting clean"
-rm -rf tmp src
+rm -rf tmp src/ios src/anrdroid
```
Comment (bhariharan) [inline tools/update.sh]: Typo on ```android```.

Comment (khawkins) [inline tools/update.sh]: doh!

**Comment by bhariharan** [comment]
:+1:

**Comment by kchitalia** [comment]
:+1:

**Pull request merged**